data/Templates/Ccda/Entry/AllergiesAndAdverseReaction/_entry_act_entryRelationship.liquid

```diff
@@ -1,4 +1,4 @@
-{% if relationship.observation.templateId | to_json_string | contains: '2.16.840.1.113883.10.20.22.4.7' -%}
+{% if relationship.observation.templateId | to_json_string | contains: '"2.16.840.1.113883.10.20.22.4.7"' -%}
```

Comment:
What's the consideration that we add double quotation marks here?

Reply:
If we don't add a double quotation here, a string contains this substring may also be included. For example, we could like to match "2.16.840.1.113883.10.20.22.4.7" here, if we don't add a double quotation, another id "2.16.840.1.113883.10.20.22.4.71"  will also match this condition.

Reply:
Got it, we have to use double quotation since we are using `contains` filter.